src/components/Data/Realtime/SubscriptionControls.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="valign-center">
     <div class="col s5 m5 l4">
-      <button
+      <!-- <button
```

Comment:
maybe you forgot to clean here, or you don't want to lose this precious code ? ;)

Reply:
I'd rather delete the whole component, since we use it no more...